examples/mssqlclient.py

```diff
@@ -40,6 +40,7 @@
                                                                                   'Authentication (default False)')
     parser.add_argument('-debug', action='store_true', help='Turn DEBUG output ON')
     parser.add_argument('-show', action='store_true', help='show the queries')
+    parser.add_argument('-cmd', action='extend', nargs='+', help='Commands to execute in the SQL shell. Multiple commands can be passed.')
```

Comment:
Changed parameter name and `nargs` value

```suggestion
    parser.add_argument('-command', action='extend', nargs='*', help='Commands to execute in the SQL shell. Multiple commands can be passed.')
```